test/coord.jl

```diff
@@ -0,0 +1,10 @@
+using DimensionalData, Test
+
+dim = Coord([(1.0,1.0,1.0), (1.0,2.0,2.0), (3.0,4.0,4.0), (1.0,3.0,4.0)], (X, Y, Z))
+da = DimArray(0.1:0.1:0.4, dim)
+
+@test da[Coord()] === da
```

Comment:
```suggestion
@test da[Coord()] === da
@test da[Coord([1, 2])] == [(1.0,1.0,1.0), (1.0,2.0,2.0)]
```
Just add a test for safety of indexing via `Coord` without any of the selectors

Reply:
Good idea, but maybe you mean:

```julia
da[Coord([1, 2])] == [0.1, 0.2]
```